PerformanceTests/Sources/PerformanceTestScript/Util/Benchmark.swift

```diff
@@ -0,0 +1,69 @@
+//
+// Copyright Amazon.com Inc. or its affiliates.
+// All Rights Reserved.
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+
+import Foundation
+
+struct BenchmarkResult: Codable {
```

Comment:
I think using `PerformanceXYZ` throughout the entire module would be easier to digest, instead of using `Performance` and `Benchmark` interchangeably . E.g., this could be named `PerformanceTestResult`.